Libraries/WNScripting/src/convert_struct_to_ast.cpp

```diff
@@ -96,7 +102,39 @@ ast_type* parse_ast_convertor::convertor_context::walk_struct_definition(
   containers::deque<memory::unique_ptr<declaration>>
       constructor_order_declarations(m_allocator);
 
+  // Either all or none of the chain must be synchronized.
+  if (_def->is_synchronized()) {
+    synchronized_decl_types =
+        memory::make_unique<struct_definition>(m_allocator, m_allocator,
+            (containers::string(m_allocator, "__") +
+                _def->get_name().to_string(m_allocator) + "__actor_data")
+                .c_str(),
+            false, nullptr);
+    synchronized_decl_types->copy_location_from(_def);
+    if (!m_type_manager->register_struct_definition(
+            synchronized_decl_types.get())) {
+      synchronized_decl_types->log_line(m_log, logging::log_level::error);
+      return false;
```

Comment:
Build doesn't like this. Looks like this functions return type is `ast_type*` so should this be a `nullptr`?